leap_c/torch/rl/sac.py

```diff
@@ -282,7 +278,8 @@ def train_loop(self) -> Iterator[int]:
     def act(
         self, obs, deterministic: bool = False, state=None
     ) -> tuple[np.ndarray, None, dict[str, float]]:
-        obs = self.task.collate([obs], self.device)
+        # TODO (Jasper): Update collate functionality
```

Comment:
There have been a few updates on the collate functionality, so not sure if this is a leftover TODO